### DISCUSSION TITLE
Not Working in Mac or Google Calendar, but working in MS Outlook!

### DISCUSSION THREAD

**Discussion by sladeindustries** [spatie/icalendar-generator#108] [Q&A]
Hi,

This link is working in MS Calendar, but mac and google calendar only displays the first event in the calendar. I ran the URL using icalvalidator which you can access it here.

Permalink: https://icalendar.org/validator.html?url=https://tarong.simplexslade.cloud/admin/ical-calendar/908887

and it shows the following error, could it be because of this?

`
Problem! Found 1 error
Errors
Lines not delimited by CRLF sequence [near line # 1]
Reference: [RFC 5545 3.1. Content Lines](http://icalendar.org/iCalendar-RFC-5545/3-1-content-lines.html)
`

The calendar code is as below...
` 
$icalendarevents[$model->{$source['field']}] = Event::create()
                                                                ->name(Tag::where('id', $model->{$source['field']})->value('tag_name'))
                                                                    ->uniqueIdentifier(Tag::where('id', $model->{$source['field']})->value('tag_name'))
                                                                    ->startsAt(Carbon::parse($crudFieldValue))
                                                                    ->url(route($source['route'], $model->id))
                                                                    ->withoutTimezone();

//Adding the events to the calendar.

$icalendar[]=Calendar::create('SiEx HAC Calendar')->withoutAutoTimezoneComponents()->event([$icalendarevents[$model->{$source['field']}]])->get();
`

**Comment by rubenvanassche** [comment]
The current, link shows no problems, so I'll guess the issue is fixed?

First time I looked at this issue the contents of the ical file had multiple calendars in it which is not allowed if I'm not mistaken.
